src/main/java/com/kodilla/ecommercee/controller/GroupController.java

```diff
@@ -1,4 +1,33 @@
 package com.kodilla.ecommercee.controller;
 
+import com.kodilla.ecommercee.domain.dto.GroupDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.ArrayList;
+
+@RestController
+@RequestMapping("/v1/shop/groups")
+@RequiredArgsConstructor
 public class GroupController {
```

Comment:
W GET (by Id) i w PUT chyba jeszcze to productId też powinno być przy tworzeniu GroupDto.